README.md

```diff
@@ -141,6 +141,16 @@ from your bash compatible shell.  This does the following things:
 * Runs all of our [phpunit](https://phpunit.de/) unit tests.
 * Performs static analysis with [Phan](https://github.com/phan/phan), [Psalm](https://psalm.dev/) and [PHPStan](https://phpstan.org/user-guide/getting-started)
 
+### Other PHP versions
+
+We current try to support PHP 7.4, 8.0 and 8.1 - the developer image `ghcr.io/open-telemetry/opentelemetry-php/opentelemetry-php-base` is tagged as `7.4`, `8.0` and `8.1` respectively, with `7.4` being the default.
+You can simulate the important github actions against other PHP versions by running the following command:
```

Comment:
Could you phrase this differently? The github actions are (luckily) not important at all, they merely excute our QA checks, which could be done by any other CI system.